Monika After Story/game/script-greetings.rpy

```diff
@@ -3832,3 +3833,40 @@ label greeting_code_help_outro:
 
     $ mas_lockEVL("greeting_code_help", "GRE")
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.greeting_database,
+            eventlabel="greeting_back_from_store",
+            unlocked=True,
+            category=[store.mas_greetings.TYPE_STORE],
+        ),
+        code="GRE"
+    )
+
+label greeting_back_from_store:
+    if mas_isMoniNormal(higher=True):
+        m 1hua "Welcome back, [player]!"
+        m 1hub "Did you get what you needed from the store?"
+        m 2eua "I hope you chose healthy food if you bought any."
+        m 2hua "Let's spend some more time together~"
+
+    elif mas_isMoniUpset():
+        m 2esc "Oh,{w=0.2} you're back."
+        m 2rsc "I hope you got everything you needed."
+        m 2wud "{cps=*2}Maybe might be in a better mood too{/cps}{nw}"
```

Comment:
Do you think we should rework this phrase so that it has a subject? I'm thinking:

m 2wud "{cps=*2}Hopefully you're in a better mood now too.{/cps}{nw}"

Reply:
Sure. Probably change the expression to `2rud` too